### DISCUSSION TITLE
I need multi lines editor for text fields.

### DISCUSSION THREAD

**Discussion by sergeynilov** [dbeaver/dbeaver#14466] [Q&A] [answered]
Hello,
With Version 21.2.1 I wonder if there is a way for text fields open some kind of multylines editor?

Maybe like WYSIWYG type ?

Thanks!

**Comment by fabianMendez** [comment]
Hi,

You can bring a multiline text editor by right clicking on the cell and selecting `Edit > Edit cell` or with the keyboard shortcut `Shift + Enter`.

**Comment by ShadelessFox** [comment, marked as answer]
Hello @sergeynilov,

Besides the ways for value editing described above, you can also use value panel.

https://github.com/dbeaver/dbeaver/wiki/Panels#value-viewer
